tests/reader.rs

```diff
@@ -0,0 +1,348 @@
+use osmpbfreader::objects::{Node, NodeId, OsmId, Ref, Relation, RelationId, Tags, Way, WayId};
+use std::fs::File;
+
+#[derive(Debug, Default, PartialEq)]
+struct Stats {
+    num_nodes: i64,
+    num_ways: i64,
+    num_relations: i64,
+    sum_nodes_ids: i64,
+    sum_ways_ids: i64,
+    sum_relations_ids: i64,
+}
+
+const PBF: &str = "tests/clipperton.osm.pbf";
+const PBF_STATS_ALL: Stats = Stats {
+    num_nodes: 3097,
+    num_ways: 151,
+    num_relations: 17,
+    sum_nodes_ids: 22498900142500,
+    sum_ways_ids: 155207265285,
+    sum_relations_ids: 112821863,
+};
+const PBF_STATS_HIGHWAY: Stats = Stats {
+    num_nodes: 9,
+    num_ways: 3,
+    num_relations: 0,
+    sum_nodes_ids: 114394331047,
+    sum_ways_ids: 4118264166,
+    sum_relations_ids: 0,
+};
+
+fn check_items() -> (Vec<Node>, Vec<Way>, Vec<Relation>) {
```

Comment:
This function name is missleading: a `check` function is expected to return a bool.